pom.xml

```diff
@@ -15,7 +15,7 @@
   </scm>
   <properties>
     <java.version>1.8</java.version>
-    <activiti-cloud-dependencies.version>7.0.105</activiti-cloud-dependencies.version>
+    <activiti-cloud-dependencies.version>7.0.91</activiti-cloud-dependencies.version>
```

Comment:
@salaboy We can update this and the other PRs but not much point doing so until immediately before merging. Or we could merge and then merge an updatebot PR so that it is overwritten.